packages/shared-components/src/components.d.ts

```diff
@@ -9,11 +9,13 @@ import { DropdownItem } from "./models/dropdown/dropdown-item";
 import { DropdownItemAlignment } from "./models/dropdown/dropdown-item-alignment";
 import { ExternalMenuModel } from "./components/onto-navbar/external-menu-model";
 import { NavbarToggledEvent } from "./components/onto-navbar/navbar-toggled-event";
+import { Placement } from "tippy.js";
 import { TranslationParameter } from "./models/translation/translation-parameter";
 export { DropdownItem } from "./models/dropdown/dropdown-item";
 export { DropdownItemAlignment } from "./models/dropdown/dropdown-item-alignment";
 export { ExternalMenuModel } from "./components/onto-navbar/external-menu-model";
 export { NavbarToggledEvent } from "./components/onto-navbar/navbar-toggled-event";
+export { Placement } from "tippy.js";
```

Comment:
duplication

Reply:
This is generated code and I wonder why this might be happening

Reply:
This is very strange. I deleted the duplication, but it reappeared after rebuilding.